service/controller/app/v1/resource/status/create.go

```diff
@@ -19,7 +19,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
-	name := key.AppName(cr)
+	name := cr.GetName()
```

Comment:
AppName is the name of the app e.g. `prometheus`. We use `cr.GetName()` so we can have multiple instances of an app in a cluster e.g. `my-cool-prometheus`.

See https://github.com/giantswarm/app-operator/blob/ed5cfae10c0035db0763eded7dfd998124c46d02/service/controller/app/v1/resource/chart/desired.go#L42 for where the name is set.